Library/OcBootManagementLib/OcBootManagementLib.c

```diff
@@ -329,14 +329,11 @@ OcRunBootPicker (
         }
 
         //
-        // Do screen clearing for builtin menu here, so that it is possible to see the action.
-        // TODO: Probably remove that completely.
+        // Clear screen from previous console contents (builtin picker,
+        // or previous console tool even in Canopy) before loading the entry.
         //
+        gST->ConOut->ClearScreen (gST->ConOut);
```

Comment:
I believe doing this always is correct here - it makes OpenCanopy behave as builtin picker always behaved.

Without this, then whenever you start a console tool, or anything in console, e.g. macOS verbose boot, then the first thing you see is the previous contents of the console scrolling onto the screen, before the chosen entry starts drawing its stuff. With this, it's fine